Sofa/Component/SolidMechanics/FEM/Elastic/src/sofa/component/solidmechanics/fem/elastic/TetrahedronFEMForceField.inl

```diff
@@ -1879,7 +1879,12 @@ void TetrahedronFEMForceField<DataTypes>::drawTrianglesFromRangeOfTetrahedra(
             }
         }
 
-        // create corresponding colors
+        // create the bluish default colors
+        color[0] = sofa::type::RGBAColor(0.0, 0.0, 1.0, 1.0);
+        color[1] = sofa::type::RGBAColor(0.0, 0.5, 1.0, 1.0);
+        color[2] = sofa::type::RGBAColor(0.0, 1.0, 1.0, 1.0);
+        color[3] = sofa::type::RGBAColor(0.5, 1.0, 1.0, 1.0);
```

Comment:
```suggestion
        color[0] = sofa::type::RGBAColor(0.0f, 0.0f, 1.0f, 1.0f);
        color[1] = sofa::type::RGBAColor(0.0f, 0.5f, 1.0f, 1.0f);
        color[2] = sofa::type::RGBAColor(0.0f, 1.0f, 1.0f, 1.0f);
        color[3] = sofa::type::RGBAColor(0.5f, 1.0f, 1.0f, 1.0f);
```
because `RGBAColor` stores `float`s